schema.sql

```diff
@@ -0,0 +1,18 @@
+create or replace table wp_wporg_gp_translation_events_actions
+(
+    event_id       int(10)     not null comment 'ID of the event',
+    user_id        int(10)     not null comment 'ID of the user who authored the translation',
+    translation_id int(10)     not null comment 'ID of the translation',
+    created_at     datetime    not null comment 'When the translation was created, in UTC',
+    locale         varchar(10) not null comment 'Locale of the translation'
+)
```

Comment:
We discussed to add a primary key for `event_id` `user_id` and `translation_id` and add a `type` field.